Build/Strategic/Meanwhile.cc

```diff
@@ -438,6 +442,7 @@ static void BeginMeanwhileCallBack(MessageBoxReturnValue const bExitValue)
 	else
 	{
 		// skipped scene!
+		gfInMeanwhile = FALSE;
```

Comment:
Not setting it back sounds like a problem... I'll look into this, I used to know where the skipping happens :)